examples/tutorials/lighting/app.ts

```diff
@@ -1,6 +1,6 @@
-import {glsl, NumberArray, UniformStore, ShaderUniformType} from '@luma.gl/core';
-import {AnimationLoopTemplate, AnimationProps, Model, CubeGeometry} from '@luma.gl/engine';
-import {phongMaterial, lighting} from '@luma.gl/shadertools';
+import {glsl, NumberArray} from '@luma.gl/core';
+import {AnimationLoopTemplate, AnimationProps, Model, CubeGeometry, _ShaderInputs} from '@luma.gl/engine';
```

Comment:
I have started to think that since it already has all the shader modules, this class should not only represent inputs but in fact the entire shader. I think it could merge with the ShaderAssembler class.